test/bats/test.bats

```diff
@@ -34,7 +34,9 @@ SLEEP_TIME=1
     sleep 5
     run kubectl apply -f ./library/default/samples/constraint.yaml
     assert_success
-    sleep 5
+    sleep 5    
```

Comment:
can we add a comment specifying this is testing the status property?

Reply:
thanks , fixed!